trieTools/zeroBalanceSystemAccountChecker/tokenBalanceGetter.go

```diff
@@ -50,21 +52,29 @@ func (tbg *tokenBalanceGetter) fetchTokenBalanceWithRetrial(address, tokenID str
 			return gjson.Get(body, "data.tokenData.balance").String(), nil
 		}
 
+		ctRetrials++
+
 		log.Warn("could not get balance; retrying...",
 			"address", address,
 			"tokenID", tokenID,
 			"token nonce", nonce,
 			"error http", errHttp,
 			"error body", errBody,
 			"num retrials", ctRetrials)
-
-		ctRetrials++
 	}
 
-	return "", fmt.Errorf("could not get adress's tokens = %s after num of retrials = %d", address, maxRequestsRetrial)
+	return "", fmt.Errorf("%w; address = %s after num of retrials = %d", errCouldNotGetBalance, address, maxRequestsRetrial)
 }
 
 func (tbg *tokenBalanceGetter) getBody(response *http.Response) (string, error) {
+	if response == nil {
+		return "", fmt.Errorf("got nil http response")
```

Comment:
perhaps extract tall errors for this method to `errors.go` as well

Reply:
Extracted only for `errNilHttpResponse` and `errNilHttpResponseBody`, since other messages are wrappers to add more context for other errors